internal/service/servicefakes/fake_config_service_interface.go

```diff
@@ -0,0 +1,236 @@
+// Code generated by counterfeiter. DO NOT EDIT.
+package servicefakes
```

Comment:
fakes not sufficient package name? service/servicefakes looks incorrect

Reply:
seems to be a counterfeiter convention :/

Reply:
we were doing some weird file parsing before so I changed it to the default that counterfeiter had because it was causing issues. Hence the weird package name